.github/workflows/enforce-labels.yml

```diff
@@ -0,0 +1,15 @@
+name: Enforce PR labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
+jobs:
+  enforce-label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: yogevbd/enforce-label-action@2.1.0
+      with:
+        #REQUIRED_LABELS_ANY: "bug,enhancement,skip-changelog"
+        #REQUIRED_LABELS_ANY_DESCRIPTION: "Select at least one label ['bug','enhancement','skip-changelog']"
+        BANNED_LABELS: "test-only-fast"
```

Comment:
I dont understand what this is actually doing, preventing the user from assigning that label? Which purpose?

Reply:
Preventing merging if label is set. The test will fail and it's required.

It's so that we do not forget to remove the label before merging.